modules/wsl-distro.nix

```diff
@@ -4,6 +4,7 @@ with lib;
 
 let
   bashWrapper = pkgs.writeShellScriptBin "sh" ''
+    export PATH=${lib.makeBinPath [ pkgs.systemd pkgs.gnugrep ]}
     . ${config.system.build.etc}/etc/set-environment
```

Comment:
Doesn't this PATH get set here? Not sure where a command should be executed here. I am probably missing some context

Reply:
hmmm, I see. We should comment that.